tests/foreman/cli/test_subscription.py

```diff
@@ -244,479 +228,3 @@ def test_negative_manifest_refresh(self):
             Subscription.refresh_manifest({
                 'organization-id': self.org['id'],
             })
-
-    @tier4
-    def test_positive_restore_ak_and_content_hosts_subscriptions(self):
```

Comment:
I wonder why these test has been removed as part of hammer-csv automation tests removal ?

Cant we just remove CSV part from it and keep the entire test as it is.

Reply:
@jyejare the complete test steps are depended on `hammer-csv` hence not any worth this to keep it.

Reply:
So isn't there anything that can take place of  CSV (```I mean just replace CSV with something similar```)

Reply:
@jyejare good question ... As I have received the Bug to remove it however that does not have any answer to replace this one.
Also when I checked the features around sat 6.6 did not find anything coming as replacement.
So my ans is no (I am not sure if anything +/-).